features/IFC102_Absence-of-deprecated-entities.feature

```diff
@@ -5,33 +5,38 @@
 Feature: IFC102 - Absence of deprecated entities
 
 This rule verifies that the IFC model does not have deprecated entities, attributes or enumerators. 
+IFC2X3 : https://standards.buildingsmart.org/IFC/RELEASE/IFC2x3/TC1/HTML/deprecated_constructs.htm
+IFC4X3: https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/content/introduction.htm
 
-Background: 
-  Given A model with Schema "IFC4.3"
+  Scenario Outline: Check for deprecated entities - IFC4.3
 
-  Scenario Outline: Check for deprecated entities
-  
+    Given A model with Schema "IFC4.3"
     Given An IFC model
 
     Then There must be less than 1 instance(s) of <entity>
 
     Examples:
       | entity | 
+      | IfcBuildingElementProxy | 
       | IfcBuildingSystem | 
+      | IfcBridgePart |
       | IfcCivilElement | 
       | IfcCivilElementType | 
       | IfcElectricDistributionBoard | 
       | IfcElectricDistributionBoardType | 
       | IfcFaceBasedSurfaceModel | 
+      | IfcGeographicElementTypeEnum |
       | IfcPostalAddress | 
       | IfcRelConnectsPortToElement | 
       | IfcRelServicesBuildings | 
+      | IfcVibrationIsolator |
       | IfcTelecomAddress | 
       | IfcTrapeziumProfileDef | 
 
```

Comment:
Additions on L28 and 32 - I don't see them in the deprecated list.

Reply:
https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcMaterialList.htm

https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcMaterialClassificationRelationship.htm

https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcTextLiteral.htm

For the first two, we see the message; 
 DEPRECATED This definition will be removed in a future major release of this standard 
 
 However, regarding `IfcTextLiteral`

> The use of [IfcTextLiteral](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcTextLiteral.htm) is deprecated and it will be made abstract.

Therefore it should probably included in 
`Check for deprecated explicitly instantiated entities `